### PR TITLE
updating and adding the manifest for 2.12.0

### PR DESCRIPTION
**Pull request by Adarshkumar14**
Signed-off-by: Adarshkumar14 <adarsh.kumar@harness.io>

### CODE REVIEW

File: k8s-manifests/ci/hce-namespace.yaml [hunk 1]
```diff
@@ -975,7 +975,23 @@ metadata:
   labels:
     app: mongo
   name: mongo-service
+  namespace: litmus
```
Comment (Jonsy13) [inline k8s-manifests/ci/hce-namespace.yaml:978]: We can remove namespace here

File: k8s-manifests/ci/hce-namespace.yaml [hunk 2]
```diff
@@ -975,7 +975,23 @@ metadata:
...
+  selector:
+    component: database
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: mongo
+  name: mongo
+  namespace: litmus
```
Comment (Jonsy13) [inline k8s-manifests/ci/hce-namespace.yaml:992]: here also

File: k8s-manifests/latest/hce-namespace.yaml [hunk 3]
```diff
@@ -625,12 +629,11 @@ spec:
       labels:
         component: litmusportal-frontend
     spec:
-      automountServiceAccountToken: false
```
Comment (Jonsy13) [inline k8s-manifests/latest/hce-namespace.yaml:628]: looks like this got removed

File: k8s-manifests/latest/hce-namespace.yaml [hunk 4]
```diff
@@ -923,7 +926,6 @@ spec:
       labels:
         component: database
     spec:
-      automountServiceAccountToken: false
```
Comment (Jonsy13) [inline k8s-manifests/latest/hce-namespace.yaml:926]: here also

File: k8s-manifests/latest/hce-namespace.yaml [hunk 5]
```diff
@@ -973,7 +975,23 @@ metadata:
...
+  selector:
+    component: database
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: mongo
+  name: mongo
+  namespace: litmus
```
Comment (Jonsy13) [inline k8s-manifests/latest/hce-namespace.yaml:992]: we can remove namespace here

File: k8s-manifests/2.12.0/hce-namespace.yaml [hunk 6]
```diff
@@ -0,0 +1,999 @@
...
+          requests:
+            storage: 20Gi
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: mongo
+  name: mongo-service
+  namespace: litmus
```
Comment (Jonsy13) [inline k8s-manifests/2.12.0/hce-namespace.yaml]: remove namespace

File: k8s-manifests/2.12.0/hce-namespace.yaml [hunk 7]
```diff
@@ -0,0 +1,999 @@
...
+  selector:
+    component: database
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: mongo
+  name: mongo
+  namespace: litmus
```
Comment (Jonsy13) [inline k8s-manifests/2.12.0/hce-namespace.yaml]: remove namesace

File: k8s-manifests/latest/hce-namespace.yaml [hunk 8]
```diff
@@ -973,7 +975,23 @@ metadata:
   labels:
     app: mongo
   name: mongo-service
+  namespace: litmus
```
Comment (amityt) [inline k8s-manifests/latest/hce-namespace.yaml:978]: Do we need namespace here?

**Review by amityt — CHANGES_REQUESTED** [review]

File: hce/values.yaml [hunk 9]
```diff
@@ -255,19 +255,19 @@ portal:
     imageEnv:
       ARGO_WORKFLOW_CONTROLLER_IMAGE: "chaosnative/workflow-controller:v3.2.3"
       ARGO_WORKFLOW_EXECUTOR_IMAGE: "chaosnative/argoexec:v3.2.3"
```
Comment (Jonsy13) [inline hce/values.yaml:257]: upgrade to `v3.3.1`

File: hce/values.yaml [hunk 10]
```diff
@@ -245,29 +245,29 @@ portal:
...
     genericEnv:
       CHAOS_CENTER_UI_ENDPOINT: ""
       SELF_CLUSTER: "true"
       CONTAINER_RUNTIME_EXECUTOR: "k8sapi"
       HUB_BRANCH_NAME: "v2.11.x"
       AGENT_DEPLOYMENTS: "[\"app=chaos-exporter\", \"name=chaos-operator\", \"app=event-tracker\", \"app=workflow-controller\"]"
       REMOTE_HUB_MAX_SIZE: "5000000"
-      WORKFLOW_HELPER_IMAGE_VERSION: "2.11.0"
+      WORKFLOW_HELPER_IMAGE_VERSION: "2.12.0"
       ENTERPRISE_HUB_BRANCH_NAME: "v2.11.x"
```
Comment (Jonsy13) [inline hce/values.yaml:271]: change it to `v2.12.x`

**Review by amityt — APPROVED** [review]

**Review by Jonsy13 — APPROVED** [review]

**Pull request merged**